scripts/variantstore/wdl/GvsJointVariantCalling.wdl

```diff
@@ -0,0 +1,85 @@
+version 1.0
+
+import "GvsUnified.wdl" as GvsUnified
+
+workflow GvsJointVariantCalling {
+    input {
+        String dataset_name
+        String project_id
+        Array[String] external_sample_names
+        Array[File] input_vcfs
+        Array[File] input_vcf_indexes
+        String callset_identifier
+        String? extract_output_gcs_dir
+    }
+
+    String extract_output_file_base_name = sub(callset_identifier, "\s+|\_+", "-")
+    String extract_table_prefix = sub(callset_identifier, "\s+|\_+", "-")
+    if(false) {
```

Comment:
yes! it's because I'm obnoxiously asking the user to come up with a name that I will use for the filter set name, and the prepare table names and the exported vcf / interval lists names, so I dont want any spaces

Reply:
i'll add a comment to the code too (!)